tests/model_fields/models.py

```diff
@@ -411,6 +411,12 @@ class Meta:
         required_db_features = {"supports_json_field"}
 
 
+class GfkModel(models.Model):
+    object_id = models.PositiveIntegerField()
+    content_type = models.ForeignKey(ContentType, models.CASCADE)
+    gfk = GenericForeignKey()
```

Comment:
Needed for the below `GenericRelation` to have something to refer to, otherwise the `contenttypes.E004` check fails.